### ISSUE TITLE
How to call plugin functions

### ISSUE THREAD

**Issue by vivek1729** [flimshaw/Valiant360#25]
First of all - Great work by the devs. The plugin is awesome and I want to extend it for a small project I'm taking up. What I ultimately want to achive is play multiple videos in WebGl rendering so that I can control the x,y coordinates of all simultaneously. So the mouseover of any valiantcontainer should update the x,y of all other valiant containers appropriately. 

For that, I was just fiddling around with the code but I am not able to get a handle on the plugin object where I can call all the functions that you've defined in `Plugin.protoype`. I tried $('.valiantPhoto').Valiant360().pause() and even $('.valiantPhoto').Valiant360('pause') mentioned in the README but both seem to fail.

My question is how do I get a handle on the Valiant360 object which exposes those functions to me. I'd ulitmately want to over-ride the mouse-over functionality for it. Any help is deeply appreciated.

**Comment by e-g-b** [comment]
This seemed perfect for a prototype project I'm working but I'm having issues getting the $('.video').Valiant360('pause'); and $('.video').Valiant360('loadVideo', '/path/to/my/video.mp4'); to work propertly. They seem to fail and nothing I've done seems to work. Any help here?

**Comment by alittlebitweird** [comment]
Yeah it seems like none of the functions work unless you pass them into the initial call.
